Tools/Scripts/webkitpy/tool/commands/suggestnominations.py

```diff
@@ -96,18 +96,16 @@ def _parse_commit_message(self, commit_message):
             raise CommitLogError
 
         committer = self._contributor_from_email(committer_email)
-        if not committer:
-            raise CommitLogError
 
         commit_date_match = self._date_regexp.search(commit_message)
         if not commit_date_match:
             raise CommitLogError
         commit_date = commit_date_match.group('date')
 
-        revision_match = self._revision_regexp.search(commit_message)
-        if not revision_match:
+        canonical_match = self._canonical_regexp.search(commit_message)
+        if not canonical_match:
             raise CommitLogError
```

Comment:
I need to dig through exactly the consequences of this, but there are 2 commits in our history which don't have canonical links.

Reply:
Any thoughts about this? AFAICT this change only affects the `suggestnomiations` command. 

It seems to me that for those commits that don't have a canonical id assigned there won't be `canonical_match` and the function will exit. In other words, those commits are not considered for computing potential committer or reviewers.

Reply:
Didn't get a chance to read through the surrounding code last week, your assessment is totally correct. This is exactly the sort of "consequence" that's perfectly acceptable in this function.